tornado/speedups.c

```diff
@@ -51,20 +51,36 @@ static PyObject* websocket_mask(PyObject* self, PyObject* args) {
     return result;
 }
 
+static int speedups_exec(PyObject *module) {
+    return 0;
+}
+
 static PyMethodDef methods[] = {
     {"websocket_mask",  websocket_mask, METH_VARARGS, ""},
     {NULL, NULL, 0, NULL}
 };
 
+static PyModuleDef_Slot slots[] = {
+    {Py_mod_exec, speedups_exec},
+#if (!defined(Py_LIMITED_API) && PY_VERSION_HEX >= 0x030c0000) || Py_LIMITED_API >= 0x030c0000
+    {Py_mod_multiple_interpreters, Py_MOD_PER_INTERPRETER_GIL_SUPPORTED},
+#endif
+#if !defined(Py_LIMITED_API) && PY_VERSION_HEX >= 0x030d0000
```

Comment:
Why are these two `#if`s not the same structure? The [definitions](https://github.com/python/cpython/blob/c46635aa5a20fc1b4c5e85370fa0fa2303c47c14/Include/moduleobject.h#L76-L81) are the same. I think the second one also needs a `Py_LIMITED_API >= 0x030d0000` clause.

It would be simpler to do `#ifdef Py_mod_gil`, although I don't think it's mandated that these are `#define`s instead of global variables.

Reply:
Yeah, I'd prefer to not go with `#ifdef Py_mod_gil`. These would probably not change, but it's still more stable to do a version check.